app/models/user.rb

```diff
@@ -16,6 +16,7 @@ class User < ActiveRecord::Base
     :pto_hours_for_date_range, to: :days
 
   scope :active, -> { where(active: true) }
+  scope :time_zones, -> { pluck('DISTINCT time_zone') }
```

Comment:
Why isn't this app hooked up to CodeClimate's rubocop support? You wouldn't have to manually enforce styleguides if the PR hooks did it for you. :trollface: